pages/login.js

```diff
@@ -5,44 +5,62 @@ import Router from "next/router";
 import SkipButton from "../components/skipButton";
 import Input from "../components/formComponents/input";
 import Button from "../components/buttons/button";
+import LoginButton from "../components/buttons/loginButton";
+import { useSession } from "next-auth/react";
 
 export default function Login() {
+  const { data: session } = useSession();
+
   function handleSubmit(event) {
     event.preventDefault();
     Router.push("/dashboard");
   }
 
+  console.log(session);
+
+  if (session) {
+    setTimeout(() => {
+      Router.push("/dashboard");
+    }, 5000);
+  }
   return (
     <StyledLogin>
-      <Logo fontSize={"40px"} />
-      <Form onSubmit={handleSubmit}>
-        <Input
-          type="email"
-          id="email"
-          name="email"
-          placeholder="E-Mail*"
-          required
-          autocomplete="off"
-        />
-        <Input
-          type="password"
-          id="Password"
-          name="Password"
-          placeholder="Passwort*"
-          minLength="6"
-          required
-        />
-        <Button type="submit" label="Login">
-          Login
-        </Button>
-      </Form>
-      <CallToRegister>
-        <p>Noch kein Account?</p>
-        <Link href="/register">
-          <ToColor>Registriere dich!</ToColor>
-        </Link>
-      </CallToRegister>
-      <SkipButton />
+      {!session ? (
```

Comment:
For components in JSX logic is usually expressed in a non-negated manner. It's easier to reason when you don't have to perform acrobatics in your mind. It's a simple case here, of course, but generally speaking it's easier to reason in the affirmative case.